source/developers/how-tos/enable-translations-new-repo.rst

```diff
@@ -5,21 +5,245 @@ Enabling Translations on a New Repo
 .. contents::
  :local:
  :depth: 2
- 
-===========
-Python Repo
-===========
 
-TODO: How to enable translations infrastructure on a Python repo
+==========
+Quickstart
+==========
+
+To enable translations on a new repository
+
+- Start from an up-to-date cookiecutter (`frontend-template-application`_ for Microfrontends, `edx-cookiecutters`_
+  for Python)
+- Configure the repository according to the appropriate section later in this document.
+
+============
+Introduction
+============
+
+
+The `OEP-58 - Translations Management`_ proposal introduced new workflows and tooling to the Open edX translation
+management:
+
+- Created the new `openedx-translations project`_ in Transifex.
+- Created the new `openedx-translations repo`_ in GitHub.
+- Setup two-way links between the Transifex project and the GitHub repo via the
+  `Transifex GitHub app <https://github.com/apps/transifex-integration>`_.
+- Added new the `atlas <https://github.com/openedx/openedx-atlas>`_ command line tool to pull translations which will
+  be used in the repo Makefiles, build and deploy processes to fetch translations from the
+  `openedx-translations repo`_.
+
+After configuring the repo as shown in the sections below, the translations workflow will automatically:
+
+- Add the extracted source (English) translation files from the repository you added to the
+  `openedx-translations repo`_ in GitHub.
+- Create a new Transifex resource for your repo in the `openedx-translations project`_ in Transifex.
+- Add the translations (Arabic, French, etc) to the GitHub `openedx-translations repo`_ when translation entries are
+  updated by translators.
+
+The full background and rationale for the new workflows and tooling can be found in the
+`OEP-58 - Translations Management`_ docs.
+
```

Comment:
This section is useful context but I think belongs in a `concepts` doc probably one titled `Translation Technology and Process`.  Then I think you could add a link to that in a `..seealso` section of this document.

Reply:
I've removed this section and focused on the How-to part.

This section is a summary of the OEP-58 document. So if we're aiming for brevity, a link should suffice.

I have the habit of summerization from the time that I used to write some academic research.